samza-core/src/main/java/org/apache/samza/system/RemoteExecutionEnvironment.java

```diff
@@ -24,7 +24,11 @@
 /**
  * This class implements the {@link ExecutionEnvironment} that runs the applications in YARN environment
```

Comment:
nit: I am guessing this applies to any execution environment where job needs to be submitted to a central location. Perhaps you can call out that we only support Yarn environment today